<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalSettings</class>
 <widget class="QWidget" name="generalSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>123456</width>
    <height>123456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QGroupBox" name="appearanceGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>189</height>
    </size>
   </property>
   <property name="title">
    <string>Appearance</string>
   </property>
   <widget class="QComboBox" name="gaugeTheme">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Type of gauge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Theme</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gaugeLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>A label for the gauge (optional)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gaugeType">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Type of gauge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauge Type</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPlasmaBg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Plasmoid Background</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showRaw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Raw Value</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showIcon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Icon</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gaugeIcon">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="showAnimations">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Animations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Font</string>
    </property>
   </widget>
   <widget class="KFontRequester" name="fontChooser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="KColorButton" name="fontColor">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>60</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rangeGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Gauge Range</string>
   </property>
   <widget class="QLineEdit" name="rangeMin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The value at which the gauge reads the lowest</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rangeMax">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The value at which the gauge reads the highest&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatically Determine Range</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gaugeLabel</tabstop>
  <tabstop>gaugeTheme</tabstop>
  <tabstop>gaugeType</tabstop>
  <tabstop>showRaw</tabstop>
  <tabstop>showIcon</tabstop>
  <tabstop>gaugeIcon</tabstop>
  <tabstop>showAnimations</tabstop>
  <tabstop>showPlasmaBg</tabstop>
  <tabstop>autoRange</tabstop>
  <tabstop>rangeMin</tabstop>
  <tabstop>rangeMax</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
